lib/tasks_private/scvmm.rake

```diff
@@ -0,0 +1,54 @@
+require 'winrm'
+require 'json'
+require 'yaml'
+
+namespace :manageiq do
+  namespace :providers do
+    namespace :scvmm do
+      desc "Regenerate VCR cassette"
+      task :regenerate_cassette, [:username, :password, :host, :port] do |_t, args|
+        secrets = Rails.application.secrets.scvmm
```

Comment:
I see you have the same call in azure. Does this work? Where do  the secrets come from? `spec/mangeiq/config/secrets.yml` ? Or does `<gem_root>/config/secrets.yml` work too?